cmd/containerd-shim-runhcs-v1/pod.go

```diff
@@ -141,37 +159,35 @@ func createPod(ctx context.Context, events publisher, req *task.CreateTaskReques
 		id:     req.ID,
 		host:   parent,
 	}
+
+	if isTemplateCreation {
+		p.isATemplate = true
+	} else if templateID != "" {
+		p.isAClone = true
+	}
```

Comment:
Can we not just automatically set these based on the annotation parsing that we did earlier?